components/common/ConnectWallet/WalletAsset.vue

```diff
@@ -83,17 +86,14 @@ const backToWallet = () => {
   emit('back')
 }
 
-const isSnekOrBsx = computed(
-  () => urlPrefix.value === 'snek' || urlPrefix.value === 'bsx'
-)
 const setTotalValue = (value: number) => {
   totalValue.value = value
 }
 
-onMounted(() => {
+onMounted(async () => {
   if (identityStore.getAuthAddress) {
     console.log('fetching balance...')
-    identityStore.fetchBalance({
+    await identityStore.fetchBalance({
```

Comment:
It seems there is no need to add `async` & `await` because there is nothing else to be executed after this fetching.